### ISSUE TITLE
Wrong output dimension of the embedding_lookup table

### ISSUE THREAD

**Issue by saurabhkumar8112** [TobiasLee/Text-Classification#11]
I looked up again and again and the dimension of the output of tf.embedding_lookup should be [?,256,128], but you have written it as [?,256,100](in comments) either I couldn't understand the concept or there's something really wrong there. Please clarify.

**Comment by TobiasLee** [comment]
Hi, the dimension in comments is a legacy version of the model using pre-trained word embedding with 100 dim, so the dim is `[batch_size, seq_len, word_embedding]`. You can ignore it!
